src/views/pages/_package.html

```diff
@@ -84,6 +90,28 @@
 					this.set({ versionS: true, versionP: value });
 				},
 			},
+			microdata: {
+				get () {
+					return {
+						'@context': 'https://schema.org',
+						'@type': 'SoftwareSourceCode',
+						'name': this.get('name'),
+						'version': this.get('versionP'),
+						'description': this.get('description'),
+						'url': `https://www.jsdelivr.com/package/npm/${this.get('name')}`,
+						'codeRepository': this.get('package.repository.url'),
+						'dateCreated': Object.values(this.get('package.versions')).at(0),
+						'dateModified': Object.values(this.get('package.versions')).at(-1),
+						'programmingLanguage': 'JavaScript',
+						'license': this.get('package.license'),
```

Comment:
Looks good, except I think this one is invalid because it's supposed to be CreativeWork or URL, not a string.

Reply:
Does package have access to the Github license URL?

Reply:
No, probably just remove it.